src/mmw/package.json

```diff
@@ -29,6 +29,7 @@
     "jstify": "0.9.0",
     "leaflet": "0.7.3",
     "leaflet-draw": "0.2.3",
+    "leaflet.locatecontrol": "0.43.0",
```

Comment:
In addition to updating `package.json`, can you please update the associated shrinkwrap file using the steps outlined [here](https://docs.npmjs.com/cli/shrinkwrap#building-shrinkwrapped-packages)?